austation/code/game/objects/items/storage/belt.dm

```diff
@@ -0,0 +1,17 @@
+
+
+
+// -------------------------------------
+//     Bluespace Belt
+// -------------------------------------
+/obj/item/storage/belt/bluespace
+	name = "Belt of Holding"
+	desc = "The greatest in pants-supporting technology."
+	icon_state = "holdingbelt"
+	item_state = "holdingbelt"
```

Comment:
You've got two icon_state definitions.
Also, you need to add a `icon = 'path'` var pointing to the icon file you added